README.md

```diff
@@ -42,6 +44,12 @@ Use `fabricrc.sample` to update or initialize Fabric configuration, for example:
 $ cp fabricrc.sample ~/.fabricrc
 ```
 
+You may need to start ssh-agent:
+``` bash
+$ eval `ssh-agent`
+$ ssh-add ~/.ssh/id_rsa
+```
+
```

Comment:
This should be added to the troubleshooting section of the troubleshooting wiki page.